### PR TITLE
Oe/86bz12uch/login stepper v1

### PR DESCRIPTION
**Pull request by guardian-001**
This PR presents the first version of the login stepper for suppliers

### CODE REVIEW

File: src/assets/icons/archimatch/user-img.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import type { SvgProps } from 'react-native-svg';
+import Svg, { ClipPath, Defs, G, Rect } from 'react-native-svg';
+
+export const UserImg = (props: SvgProps) => (
+  <Svg width={32} height={32} viewBox="0 0 32 32" fill="none" {...props}>
```
Comment (MohamedMegdich1) [inline src/assets/icons/archimatch/user-img.tsx:5]: don't fix the height and the width

File: src/modules/supplier/other/creation-form-password-supplier.tsx
Comment (MohamedMegdich1) [inline src/modules/supplier/other/creation-form-password-supplier.tsx:1]: remove it if you don't need it

**Review by MohamedMegdich1 — CHANGES_REQUESTED** [review]

File: src/assets/icons/archimatch/user-img.tsx (hunk 1 above)
Comment (guardian-001) [reply src/assets/icons/archimatch/user-img.tsx:5]: Fixed

File: src/modules/supplier/other/creation-form-password-supplier.tsx
Comment (guardian-001) [reply src/modules/supplier/other/creation-form-password-supplier.tsx:1]: I will need this file in next steps

**Pull request merged**